src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -168,6 +168,12 @@ def create_feedback_issue(event, project_id, source: FeedbackCreationSource):
     if not project.flags.has_feedbacks:
         first_feedback_received.send_robust(project=project, sender=Project)
 
+    if (
+        source is not FeedbackCreationSource.CRASH_REPORT_EMBED_FORM
```

Comment:
i would modify this to be either only checking that it is `NEW_FEEDBACK_ENVELOPE` instead, as many legacy users using old APIs also send feedbacks via the user report API, or envelope.